model/pguser_test.go

```diff
@@ -0,0 +1,200 @@
+// Package model provides user-manager specific data structures,
+// which are meant to be used across the whole application.
+package model
+
+import (
+	"database/sql/driver"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewUsersRepo(t *testing.T) {
+
+	db, _, err := sqlmock.New()
+	if err != nil {
+		t.Fatalf("an error '%s' was not expected when opening a stub database connection", err)
+	}
+	defer db.Close()
+	userRepo := NewUsersRepo(db)
+
+	assert.Equal(t, db, userRepo.db)
+}
+
+func TestAdd(t *testing.T) {
+	db, mock, err := sqlmock.New()
+	if err != nil {
+		t.Fatalf("an error '%s' was not expected when opening a stub database connection", err)
+	}
```

Comment:
It is possible to change it, using `require.NoError()`

Reply:
No. Because it sqlmock database, if it is not created, we cann't continue test